build_tools/python/e2e_test_framework/models/torch_models.py

```diff
@@ -166,6 +166,97 @@
     entry_function="forward",
     input_types=["1280x384xi64", "1280x384xi64"])
 
+# FP16 Versions
+BERT_LARGE_1X384_FP16_TORCH = common_definitions.Model(
```

Comment:
(Optional): I think we can actually write code to generate configs for different batch sizes as they are identical in many places. Just suggestions, don't need to do it in this PR : )